docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -72,6 +72,11 @@ Upload the workflow to GitHub
 
 These steps are outlined `here. <https://docs.github.com/en/github/managing-files-in-a-repository/adding-a-file-to-a-repository>`__
 
+Add Test Parameter File
+-----------------------
+
+You can add a test :ref:`dict parameter file` to your source code repository, allowing other users to easily launch a test run of your workflow.  Follow `these instructions <https://usegalaxy.org/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#testing-the-workflow-recommended>`__ to create a test parameter file.  Then, add the file you have created to your GitHub repository as you did with your '.ga' Galaxy workflow file above.
```

Comment:
Was like this already, but is it possible to steer users toward the newer .yaml Galaxy workflow language? https://github.com/galaxyproject/gxformat2

(more human readable, writeable, etc. )

Reply:
Done, I basically added a "Note" at the end of the section that explains how to export, explaining "Format 2", and that if the interface allows it, you should export your workflow as a "Format 2" file.